<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winners</class>
 <widget class="QMainWindow" name="winners">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>261</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="medal_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="medal_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="medal_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;Marker Felt&quot;;
color: rgb(187, 45, 35);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;Marker Felt&quot;;
color: rgb(78, 147, 49);
</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;Marker Felt&quot;;
color: rgb(0, 38, 241);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;Marker Felt&quot;;
color: rgb(242, 232, 78);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="score_1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;Marker Felt&quot;;
color: rgb(187, 45, 35);</string>
     </property>
     <property name="text">
      <string>score</string>
     </property>
    </widget>
    <widget class="QLabel" name="score_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;Marker Felt&quot;;
color: rgb(78, 147, 49);</string>
     </property>
     <property name="text">
      <string>score</string>
     </property>
    </widget>
    <widget class="QLabel" name="score_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;Marker Felt&quot;;
color: rgb(0, 38, 241);</string>
     </property>
     <property name="text">
      <string>score</string>
     </property>
    </widget>
    <widget class="QLabel" name="score_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;Marker Felt&quot;;
color: rgb(242, 232, 78);</string>
     </property>
     <property name="text">
      <string>score</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>414</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
